README-OTEL.md

```diff
@@ -1,2 +1,104 @@
+
+![vimdiff Output - InstrumentdFunction v OtelFunction](./img/otel/otel-function-vimdiff-output.png)
+
+Let's invoke `OtelFunction` in the AWS cloud with the following command:
+
+```bash
+make remote-otel
+```
+
+Our Amazon CloudWatch Logs output and CloudWatch Metrics output should look familiar - it's similar to what we've seen when invoking `InstrumentedFunction`. However, our traces are not being sent to AWS X-Ray. Instead, they are being sent to [Honeycomb](https://honeycomb.io). Let's take a look at the generated data.
+
+![Honeycomb Home Page](./img/otel/otel-function-honeycomb-home-page.png)
+_Honeycomb Home Page_
+
+We can browse traces for the given time period to get more detail.
+
+## Browsing Traces in Honeycomb
+
+<details>
+<summary>otel-user-registration Service - otel.handler Summary</summary>
+
+![otel-user-registration Service - otel.handler Summary](./img/otel/otel-function-honeycomb-root-trace-summary.png)
+
+</details>
+
+<details>
+<summary>otel-user-registration Service - otel.handler Detail</summary>
+
+![otel-user-registration Service - otel.handler Detail](./img/otel/otel-function-honeycomb-root-trace-detail.png)
+
+</details>
+
+<details>
+<summary>otel-user-registration Service - GET Summary</summary>
+
+![otel-user-registration Service - GET Summary](./img/otel/otel-function-honeycomb-get-trace-summary.png)
+
+</details>
+
+<details>
+<summary>otel-user-registration Service - GET Detail</summary>
+
+![otel-user-registration Service - GET Detail](./img/otel/otel-function-honeycomb-get-trace-detail.png)
+
+</details>
+
+## Searching Traces in Honeycomb
+
+Similar to AWS X-Ray, we can search through our traces to get more insight into what our function is doing.
+
+<details>
+<summary>Search - http.url</summary>
+
+![Simple Search - http.url](./img/otel/otel-function-honeycomb-search-http-url.png)
+
+</details>
+
+<details>
+<summary>Search - http.url and library.version</summary>
+
+![otel-user-registration Service - GET Detail](./img/otel/otel-function-honeycomb-search-http-url-and-library-version.png)
+
+</details>
+
+### How Did We Do That?
+
+Our code has not been instrumented with the Python OTEL SDK so how did the trace data make it to Honeycomb? 
+
+_The [AWS Distro for OpenTelemetry Lambda (ADOT)](https://aws-otel.github.io/docs/getting-started/lambda) layer!_
+
+There are a few addition items to consider but the capability to use zero-code instrumentation begins with the capabilities baked into the layer. The layer includes both OpenTelemetry SDK and the ADOT Collector components. Configuration values for the collector are defined both as [environment variables in the template.yaml](./template.yaml#L139) file and the [otel-config.yaml](./function/otel/otel-config.yaml) files. In our example, we are storing the Honeycomb API Key and Honeycomb API endpoint as environment variables stored in [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).
```

Comment:
This is a really key point, maybe bold that we are storing the API Key and endpoint in the AWS Secrets Manager.